File: eBarberShop/eBarberShop.Services/Database/Uposlenik.cs
namespace eBarberShop.Services.Database
{
    public partial class Uposlenik
    {
        public int UposlenikId { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string KontaktTelefon { get; set; }
        public string? Email { get; set; }
        public string? Adresa { get; set; }

        public virtual ICollection<Rezervacija> Rezervacije { get; set; } = new List<Rezervacija>();
        public virtual ICollection<Ocjene> Ocjene { get; set; } = new List<Ocjene>();

    }
}

File: eBarberShop/UI/ebarbershop_desktop/lib/models/uposlenik/uposlenik.dart
import 'package:json_annotation/json_annotation.dart';

part 'uposlenik.g.dart';

@JsonSerializable()
class Uposlenik {
  int uposlenikId;
  String ime;
  String prezime;
  String kontaktTelefon;
  String? email;
  String? adresa;

  Uposlenik(this.uposlenikId, this.ime, this.prezime, this.kontaktTelefon, this.email, this.adresa);

  factory Uposlenik.fromJson(Map<String, dynamic> json) =>
      _$UposlenikFromJson(json);

  Map<String, dynamic> toJson() => _$UposlenikToJson(this);

}